packages/transformer/src/test/standalone/IModelTransformer.test.ts

```diff
@@ -3183,51 +3184,57 @@ describe("IModelTransformer", () => {
     targetDb.close();
   });
 
-  it("should not update root elements when skipPropagateChangesToRootElements is set to true", async () => {
-    const iModelShared: SnapshotDb =
-      IModelTransformerTestUtils.createSharedIModel(outputDir, ["A", "B"]);
-    const iModelA: SnapshotDb = IModelTransformerTestUtils.createTeamIModel(
-      outputDir,
-      "A",
-      Point3d.create(0, 0, 0),
-      ColorDef.green
-    );
-    IModelTransformerTestUtils.assertTeamIModelContents(iModelA, "A");
-    const iModelExporterA = new IModelExporter(iModelA);
+  for (const skipPropagateChangesToRootElementsTrueByDefault of [true, false]) {
+    it("should not update root elements when skipPropagateChangesToRootElements is set to true", async () => {
```

Comment:
the name of the test should be modified based off of the value of skipPropagateChangesToRootElementsTrueByDefault

You can use a `` string and do some conditionals

Reply:
resolved in [commit adbbdd68467f8525f12e36087c0b013e68e88862](https://github.com/iTwin/imodel-transformer/pull/188/commits/adbbdd68467f8525f12e36087c0b013e68e88862)